src/views/virtualmachines/list/components/VirtualMachineRow/VirtualMachineRowLayout.tsx

```diff
@@ -13,6 +14,13 @@ import { VMStatusConditionLabelList } from '../VMStatusConditionLabel';
 
 import './virtual-machine-row-layout.scss';
 
+export const isLiveMigratable = (vm: V1VirtualMachine, isSingleNodeCluster: boolean): boolean =>
```

Comment:
can you please move this function to a util file? 
this function can also be used in VirtualMachineActionFactory.tsx file it will be nice if you use it there too :)

Reply:
Yeah, I was not sure where exactly to put this function. Thanks! :)